olivetti.el

```diff
@@ -1,9 +1,14 @@
 ;;; olivetti.el --- Minor mode for a nice writing environment -*- lexical-binding: t; -*-
 
-;; Copyright (c) 2014-2016 Paul Rankin
+;; Copyright (c) 2014-2017 Paul Rankin
 
 ;; Author: Paul Rankin <hello@paulwrankin.com>
 ;; Keywords: wp
+;; Version: 1.5.8
+;; Package-Requires: ((emacs "24.4"))
+;; URL: https://github.com/rnkn/olivetti
```

Comment:
In theory there’s a homepage https://rnkn.githib.io/olivetti
But I wonder if that page is more confusing than it is useful?

Reply:
Honestly, I don't know...  The convention for ELPA packages seems to be to use the github.org/user/project page directly, but I'm pretty sure that's just a convention.  The user.github.io/project page looks very clean and user-oriented :-)  For the Debian package, I can use the github.org page as the Source, and rnkn.github.io/olivetti as the homepage if you'd like.

Reply:
Okay yep considering the confusion just between you and I about which to link to I think it’s just confusing in general, so good to go with the github.com link.